larq/optimizers_test.py

```diff
@@ -1,6 +1,5 @@
 import pytest
 import numpy as np
-import tensorflow as tf
```

Comment:
Why this change? In most of the codebase we make calls to `tf.keras.something`, not `keras.something`, right?

Reply:
> Why this change?

No reason, mainly caused by some careless early morning debugging. 👍 Reverted the change